packages/react/src/components/Footer/LegalNav.js

```diff
@@ -73,7 +75,8 @@ function renderListItems(links) {
     );
   });
 
-  renderedLinks.push(renderTrusteItem());
+  const key = parseFloat(renderedLinks[renderedLinks.length - 1].key) + 1;
```

Comment:
Given the `.map()` loop above has `key={index}` and we seem to be trying to do `lastIndex + 1`, I guess we want below. My apologies if I'm missing anything!

```suggestion
  const key = renderedLinks.length + 1;
```